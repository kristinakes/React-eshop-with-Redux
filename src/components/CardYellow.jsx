function CardYellow({ icon, title, text }) {
  return (
    <div className="bg-yellow flex flex-col items-center justify-evenly shadow-lg text-opacity-25 rounded p-5 md:px-10 h-[330px] w-[310px] ">
      <img src={icon} alt="" className="h-[46px] w-[52px]" />
      <h5 className="font-bold text-2xl capitalize leading-7 text-black-900">
        {title}
      </h5>
      <p className="font-normal text-center text-xl leading-6">{text}</p>
    </div>
  );
}
export default CardYellow;
