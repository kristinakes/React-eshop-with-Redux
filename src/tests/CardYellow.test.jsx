import renderer from "react-test-renderer";

import CardYellow from "../components/CardYellow";

it("renders correctly when there is no features", () => {
    const tree = renderer.create(<CardYellow />).toJSON();
    expect(tree).toMatchSnapshot();
})

it("renders correctly when there is a single feature", () => {
  const feature = [{icon: "wifi",
      title: "Wireless Freedom",
      text: "wireless gadgets that provide freedom of movement while using them",}];
  const tree = renderer.create(<CardYellow {...feature}/>).toJSON();
  expect(tree).toMatchSnapshot();
});