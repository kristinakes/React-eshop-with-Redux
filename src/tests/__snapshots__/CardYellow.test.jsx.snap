// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders correctly when there is a single feature 1`] = `
<div
  className="bg-yellow flex flex-col items-center justify-evenly shadow-lg text-opacity-25 rounded p-5 md:px-10 h-[330px] w-[310px] "
>
  <img
    alt=""
    className="h-[46px] w-[52px]"
  />
  <h5
    className="font-bold text-2xl capitalize leading-7 text-black-900"
  />
  <p
    className="font-normal text-center text-xl leading-6"
  />
</div>
`;

exports[`renders correctly when there is no features 1`] = `
<div
  className="bg-yellow flex flex-col items-center justify-evenly shadow-lg text-opacity-25 rounded p-5 md:px-10 h-[330px] w-[310px] "
>
  <img
    alt=""
    className="h-[46px] w-[52px]"
  />
  <h5
    className="font-bold text-2xl capitalize leading-7 text-black-900"
  />
  <p
    className="font-normal text-center text-xl leading-6"
  />
</div>
`;
